### PR TITLE
增加构建详情的查看配置项，在构建详情界面点击插件时，默认进入的是 日志 or 配置 Tab页面。 #10808

### PR DESCRIPTION
**Pull request by useryuyu**
#10808

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=10884) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=10884) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: useryuyu<br/>:x: v_yjjiaoyu<br/><hr/>**v_yjjiaoyu** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-ci?pullRequest=10884) it.</sub>

File: src/frontend/devops-pipeline/src/views/subpages/ExecDetail.vue [hunk 1]
```diff
@@ -132,6 +132,7 @@
                 <plugin
                     :exec-detail="execDetail"
                     :editing-element-pos="editingElementPos"
+                    :properties="['CONFIG', 'LOG', 'ARTIFACT']"
```
Comment (lockiechen) [inline src/frontend/devops-pipeline/src/views/subpages/ExecDetail.vue:135]: 数组，对象这种不要直接写在属性里面，这样每次diff都会导致Props不一致， 用一个computed属性存起来

File: src/frontend/devops-pipeline/src/components/ExecDetail/plugin.vue [hunk 2]
```diff
@@ -43,7 +43,7 @@
                 v-if="currentTab === 'log'"
             />
             <component
-                v-if="currentTab === key"
+                v-show="currentTab === key"
```
Comment (lockiechen) [inline src/frontend/devops-pipeline/src/components/ExecDetail/plugin.vue:46]: 这里用v-show，实际组件还是会渲染，有点浪费

File: src/frontend/devops-pipeline/src/components/ExecDetail/plugin.vue [hunk 3]
```diff
@@ -179,10 +184,24 @@
...
                     if (!tab.show) {
-                        this.currentTab = 'log'
+                        this.currentTab = val[0].name
                     }
                 },
                 deep: true
+            },
+            properties: {
+                handler (newValue) {
+                    if (JSON.stringify(newValue) === JSON.stringify(['CONFIG', 'LOG', 'ARTIFACT'])) {
```
Comment (lockiechen) [inline src/frontend/devops-pipeline/src/components/ExecDetail/plugin.vue:193]: 不要用这种方式直接去比较数组是否一致，虽然这个数组长度不多，问题不大，同样的原理，这种用join性能还好一些

newValue.join('') === ['CONFIG', 'LOG', 'ARTIFACT'].join(''), 而且我看这个默认值出现多次了，可以统一用个变量存起来，也后续方便维护

**Review by lockiechen — APPROVED** [review]

File: src/frontend/devops-pipeline/src/views/subpages/ExecDetail.vue [hunk 4]
```diff
@@ -257,6 +257,7 @@
                 return ['RUNNING', 'QUEUE'].includes(this.execDetail?.status)
             },
             panels () {
+                console.log(this.execDetail, '------------------', this.curProject)
```
Comment (lockiechen) [inline src/frontend/devops-pipeline/src/views/subpages/ExecDetail.vue:260]: 这个去掉

**Pull request merged**